packages/e2store/README.md

```diff
@@ -68,44 +123,60 @@ import {
 
 const era1File = readBinaryFile(PATH_TO_ERA1_FILE)
 
-// validate era1 file
+// Validate era1 file
+// Rebuilds epoch accumulator and validates the accumulator root
 const isValid = validateERA1(era1File)
+if (!isValid) {
+  throw new Error('Invalid Era1 file')
+}
 
-// read blocks from era1 file
+// Read blocks from era1 file
 const blocks = readERA1(era1File)
 
 for await (const blockTuple of blocks) {
   const { header, body, receipts } = await parseBlockTuple(blockTuple)
   const block = blockFromTuple({ header, body })
-  console.log(block.header.number)
+  console.log(`Block number: ${block.header.number}`)
 }
 
-// reconstruct epoch accumulator
+// Reconstruct epoch accumulator
 const headerRecords = await getHeaderRecords(era1File)
 const epochAccumulator = EpochAccumulator.encode(headerRecords)
 const epochAccumulatorRoot = EpochAccumulator.merkleRoot(headerRecords)
 ```
 
-### Read Era file
+### Era
+
+Era files are used to store beacon chain data, including beacon states and blocks.
+
+#### Read Era file
 
 ```ts
-import { readBeaconState } from "@ethereumjs/e2store"
+import { readBeaconState, readBlocksFromEra } from "@ethereumjs/e2store"
 
 const eraFile = readBinaryFile(PATH_TO_ERA_FILE)
 
 // Extract BeaconState
 const state = await readBeaconState(eraFile)
-console.log(state.slot)
+console.log(`Current slot: ${state.slot}`)
 
 // Read Beacon Blocks from era file
 let count = 0
 for await (const block of readBlocksFromEra(eraFile)) {
-  console.log(block.message.slot)
+  console.log(`Block slot: ${block.message.slot}`)
   count++
   if (count > 10) break
 }
```

Comment:
Nice, can we also add all the examples from the README to a dedicated `examples` folder so that they are auto-inserted from now on when `npm run build:examples` is run?

So, the way that this works is that you add the comment line (and you can leave the example code itself in first round) from the screenshot below in the README example block(s):

![grafik](https://github.com/user-attachments/assets/e17e7bc9-334b-458e-aff8-c4c701c26cd0)

And when you then run the npm command (and code is not yet there) you can test if the stuff is inserted automatically.